Game/level.tscn

```diff
@@ -0,0 +1,24 @@
+[gd_scene load_steps=5 format=3 uid="uid://c7hqqdujukupm"]
+
+[ext_resource type="PackedScene" uid="uid://crgwrl3y2s3g6" path="res://player.tscn" id="1_jvfr3"]
+[ext_resource type="Texture2D" uid="uid://dcparl8gbox7k" path="res://Whiteboard-Vector-PNG-Photos-3289639508.png" id="1_o5ec7"]
+[ext_resource type="PackedScene" uid="uid://d4mickvsqnq5x" path="res://bellyrub.tscn" id="2_uw1pv"]
+[ext_resource type="AudioStream" uid="uid://3ucxuxrajjmu" path="res://Jungle.wav" id="3_m5y1u"]
+
+[node name="Level" type="Node2D"]
+
+[node name="Whiteboard-vector-png-photos-3289639508" type="Sprite2D" parent="."]
+position = Vector2(934, 554.5)
+scale = Vector2(2.89, 2.50225)
+texture = ExtResource("1_o5ec7")
+
+[node name="Player" parent="." instance=ExtResource("1_jvfr3")]
+position = Vector2(199, 289)
+
+[node name="bellyrub" parent="." instance=ExtResource("2_uw1pv")]
+
+[node name="AudioStreamPlayer2D" type="AudioStreamPlayer2D" parent="."]
+position = Vector2(-72, -23)
+stream = ExtResource("3_m5y1u")
+autoplay = true
+playback_type = 1
```

Comment:
I think we may want to change this to an `AudioStreamPlayer` rather than an `AudioStreamPlayer2D`, the reason being that an `AudioStreamPlayer2D` has positional sounds, but we want the background music to play in both ears equally.

You should be able to change the type in the editor from the right-click menu
![image](https://github.com/user-attachments/assets/d617ee65-fba8-4f7b-94c9-bfce7229262f)